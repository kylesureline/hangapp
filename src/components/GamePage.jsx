import React from 'react';
import HangmanSVG from './HangmanSVG.jsx';
import LetterGrid from './LetterGrid.jsx';
import Answer from './Answer.jsx';
import EndGameModal from './EndGameModal.jsx';

export const GamePage = () => {
  return (
    <div className="page page--game">
      <HangmanSVG />
      <Answer/>
      <LetterGrid />
      <EndGameModal />
    </div>
  );
};

export default GamePage;
