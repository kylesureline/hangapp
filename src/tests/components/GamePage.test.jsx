import React from 'react';
import { shallow } from 'enzyme';
import { GamePage } from '../../components/GamePage.jsx';

test('should render GamePage correctly', () => {
  const wrapper = shallow(<GamePage />);
  expect(wrapper).toMatchSnapshot();
});
