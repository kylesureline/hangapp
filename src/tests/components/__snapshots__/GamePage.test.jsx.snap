// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GamePage correctly 1`] = `
<div
  className="game-page"
>
  <Connect(HangmanSVG) />
  <Connect(Answer) />
  <Connect(LetterGrid) />
  <Connect(EndGameModal) />
</div>
`;
